independent-publisher-connectors/Philips HUE/apiProperties.json

```diff
@@ -0,0 +1,73 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "",
+          "scopes": [],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://api.meethue.com/v2/oauth2/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://api.meethue.com/v2/oauth2/token"
+            },
+            "refreshUrl": {
+              "value": "https://api.meethue.com/v2/oauth2/token"
+            }
+          }
+        }
+      },
+      "apiKey": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "HUE application key",
+          "description": "The API Key that is returned after performin PUT and POST authorization calls.",
```

Comment:
All descriptions in apiProperties.json file should be free from grammatical errors. Please review them and fix them.